index.js

```diff
@@ -36,42 +39,46 @@ var MochaParallelTests = module.exports = function (options) {
         });
 
         runner.on('end', function() {
-            customRunner.start()
+            customRunner.start();
+
             suites.forEach(function (suite) {
                 customRunner.suite(suite);
             });
+
             passes.forEach(function (test) {
                 customRunner.pass(test);
             });
+
             failures.forEach(function (test) {
                 customRunner.fail(test);
             });
-            customRunner.testEnd()
+
+            customRunner.end();
```

Comment:
calling this is needed to get

```
Options passing
    ✓ should pass all options into mocha executable (1021ms)

  1 passing (1ms)
```

in the output. Anyway, I'm not sure that breaking the order of suite-pass-fail is okay. We definetely need a test for that.